content/zh-cn/docs/reference/access-authn-authz/authentication.md

```diff
@@ -560,7 +560,7 @@ To enable the plugin, configure the following flags on the API server:
 
 | 参数 | 描述 | 示例 | 必需？ |
 | --------- | ----------- | ------- | ------- |
-| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。此值通常设置为服务的发现 URL，不含路径。例如："https://accounts.google.com" 或 "https://login.salesforce.com"。此 URL 应指向 .well-known/openid-configuration 下一层的路径。 | 如果发现 URL 是 `https://accounts.google.com/.well-known/openid-configuration`，则此值应为 `https://accounts.google.com` | 是 |
+| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。这通常是提供商的发现URL，更改为空路径。 | 如果颁发者的OIDC发现URL为`https://accounts.provider.example/.well-known/openid-configuration`，则值应为`https://accounts.provider.example` | 是 |
```

Comment:
```suggestion
| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。这通常是提供商的发现 URL，去掉路径部分。 | 如果颁发者的 OIDC 发现URL为`https://accounts.provider.example/.well-known/openid-configuration`，则值应为 `https://accounts.provider.example` | 是 |
```

Reply:
Hi @HeavenTonight, if you have some different opinions, please provide feedback. However, it's not a good practice to click `Resolve conversation` when you only accept partial suggestions and keep silent, which may cause much more time to confirm by each reviewer, and might be somewhat impolite.